corgi/tasks/errata_tool.py

```diff
@@ -108,6 +109,17 @@ def slow_load_errata(erratum_name: str, force_process: bool = False) -> None:
                 # processing the same Brew builds / errata repeatedly
                 kwargs={"save_product": False, "force_process": False},
             )
+            # We might have already processed the build from Brew, but when it gets pushed by ET
+            # container builds can get a new name. Make sure we check for that here and update the
+            # container name if so.
+            if is_container:
+                build = SoftwareBuild.objects.filter(
+                    build_id=build_id, build_type=SoftwareBuild.Type.BREW
+                ).first()
+                if build:
+                    # Let this propagate a error
+                    nvr = build.meta_attr.get("nvr")
+                    slow_update_name_for_container_from_pyxis.delay(nvr)
```

Comment:
This won't fail the slow_load_errata task if the NVR key is missing. Below will:
```suggestion
                    # Let this propagate a error
                    nvr = build.meta_attr["nvr"]
                    slow_update_name_for_container_from_pyxis.delay(nvr)
```
Or if you want the slow_load_errata task to succeed, and only the scheduled task should fail, maybe we should says that: "Let this task succeed, but fail the scheduled task if the build has no NVR"